static/sass/_pattern_search-form.scss

```diff
@@ -0,0 +1,51 @@
+@mixin canonical-search-form {
+  .p-form--search {
+    margin-top: 18px;
```

Comment:
why px? set the viewport to 1799px  and then increase to 1800px. YOu'll see the value of 1rem change from 16 to 18px, effectively zoomingm everything in by a factor of 18/16. Anything expressed in px won't scale though, so it's always best to express things in rem, and also use the appropriate spacing variables.

Reply:
I think I meant to delete that line altogether honestly, I don't remember why I was playing around with the margins originally but it isn't needed.